Mathlib/CategoryTheory/Limits/FunctorCategory/Finite.lean

```diff
@@ -5,6 +5,7 @@ Authors: Dagur Asgeirsson
 -/
 import Mathlib.CategoryTheory.Limits.FunctorCategory.Basic
 import Mathlib.CategoryTheory.Limits.Shapes.FiniteProducts
```

Comment:
```suggestion
```

Reply:
This is imported by `Preserves.Finite`

Reply:
This doesn't compile with the suggested removal